src/actions/delete.ts

```diff
@@ -102,16 +104,17 @@ export class DeleteAction {
 
     const deleteItem: ContextMenuRegistry.RegistryItem = {
       displayText: (scope) => {
+        const shortcut = getShortActionShortcut(this.deleteShortcutName);
         if (!this.oldContextMenuItem) {
-          return 'Delete block (Del)';
+          return Msg['DELETE_BLOCK'].replace('%1', shortcut);
         }
 
         type DisplayTextFn = (p1: ContextMenuRegistry.Scope) => string;
         // Use the original item's text, which is dynamic based on the number
         // of blocks that will be deleted.
         const oldDisplayText = this.oldContextMenuItem
           .displayText as DisplayTextFn;
-        return oldDisplayText(scope) + ' (Del)';
+        return oldDisplayText(scope) + ` (${shortcut})`;
```

Comment:
Will this localize for RTL correctly?

Reply:
No, but this is kind of a hack in the first place so I think that's acceptable for now.